libbulletc/CMakeLists.txt

```diff
@@ -375,6 +376,8 @@ ADD_LIBRARY(${BULLETC_LIB} SHARED
     src/btTriangleIndexVertexArray_wrap.h
     src/btTriangleIndexVertexMaterialArray_wrap.cpp
     src/btTriangleIndexVertexMaterialArray_wrap.h
+    src/btTriangleInfoMap_wrap.cpp
+    src/btTriangleInfoMap_wrap.h

```

Comment:
These are rarely used outside of Bullet internals and the rest of the wrapper doesn't relate to these classes yet.
But they are exposed publicly in Bullet, so it's OK to include the files.